_includes/header.html

```diff
@@ -1,11 +1,5 @@
-<header class="header-container">
-  <a href="/" aria-label="Savile Home">
-    <img class="logo" src="{{site.assets_url}}/turing-school-mark-gray.png" role="presentation"/>
+<header class="s-nav header-container">
+  <a class="s-nav-icon" href="/" aria-label="Home">
+    <img class="s-nav-icon" src="https://assets.turing.edu/turing-school-mark-gray.png" alt="" />
```

Comment:
~Can we put something in the `alt`, maybe `"Turing School logo"` ?~

Scratch that, for accessibility reasons it looks like the `alt` should describe the action the link takes. [Source from W3C](https://www.w3.org/WAI/tutorials/images/functional/). And then you don't need to put the `aria-label` in the surrounding `<a>` tag.

```suggestion
    <img class="s-nav-icon" src="https://assets.turing.edu/turing-school-mark-gray.png" alt="Savile Home" />
```

Reply:
Also why not use the `{{site.assets_url}}`?

Reply:
Oh interesting. I was viewing the Turing logo as a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/) so that's why I was leaving alt blank and adding the label to the anchor itself!

Reply:
For the assets link I think I'd copy and pasted and forgot that was available!